pkg/reconciler/testing/inmemorychannel.go

```diff
@@ -173,3 +175,10 @@ func WithInMemoryChannelStatusSubscribers(subscriberStatuses []duckv1alpha1.Subs
 			Subscribers: subscriberStatuses})
 	}
 }
+
+func WithInMemoryNamespaceAnnotation(imc *v1alpha1.InMemoryChannel) {
```

Comment:
nit: `SetNamespaceAnnotation`?

Reply:
Oops name is correct, implementation is not ... Thanks!